AssetInformationListener/Domain/DomainEntity.cs

```diff
@@ -0,0 +1,13 @@
+using System;
+
+namespace AssetInformationListener.Domain
+{
+    public class DomainEntity
+    {
+        // TODO - define your domain entity/entities here
```

Comment:
Remove comment as complete

Reply:
It will be when the next ticket is done. This is just setting up the application shell.